pkg/controller/pcidevice/pcidevice_controller.go

```diff
@@ -117,3 +137,13 @@ func (h Handler) reconcilePCIDevices(nodename string) error {
 
 	return nil
 }
+
+func containsString(elements []string, element string) bool {
```

Comment:
This seems like something that should be a generic standard library function, I can't find anything like that, but let's keep an eye out for it.